python/paddle/nn/quant/qat/conv.py

```diff
@@ -17,8 +17,10 @@
 from paddle.nn import Layer
 from paddle.nn import functional as F
 
+from ..format import ConvertibleQuantedLayer
 
-class QuantedConv2D(Layer):
+
+class QuantedConv2D(ConvertibleQuantedLayer):
     """
     The computational logic of QuantizedConv2D is the same with Conv2D.
     The only difference is that its inputs are all fake quantized.
```

Comment:
The computational logic of QuantizedConv2D is the same as Conv2D. The only difference is that its inputs are all fake quantized.